### PR TITLE
Add more project URLs

### LINKED ISSUE #19: Add more project URLs to the dist metadata
Similar to what https://pypi.org/project/ansible-lint/ has in the sidebar, we should also add links to CI, Codecov, CoC, MLs and issues.

### PR DESCRIPTION
**Pull request by felixfontein**
Fixes #19.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -15,6 +15,13 @@ packages = [
     { include = "tests", format = "sdist" }
 ]
 
+[tool.poetry.urls]
+"Mailing lists" = "https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information"
+"Code of Conduct" = "https://docs.ansible.com/ansible/latest/community/code_of_conduct.html"
+"CI: GitHub" = "https://github.com/ansible-community/ansible-pygments/actions?branch:main+event:push"
+"CI: CodeCov" = "https://app.codecov.io/gh/ansible-community/ansible-pygments"
+"Bug tracker" = "https://github.com/ansible-community/ansible-pygments"
```
Comment (webknjaz) [inline pyproject.toml]:
```suggestion
"Bug tracker" = "https://github.com/ansible-community/ansible-pygments/issues"
```

File: pyproject.toml [hunk 2]
```diff
@@ -15,6 +15,13 @@ packages = [
     { include = "tests", format = "sdist" }
 ]
 
+[tool.poetry.urls]
+"Mailing lists" = "https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information"
+"Code of Conduct" = "https://docs.ansible.com/ansible/latest/community/code_of_conduct.html"
```
Comment (webknjaz) [inline pyproject.toml:20]: I wonder if we could add some tracking here...

File: pyproject.toml (hunk 1 above)
Comment (felixfontein) [reply pyproject.toml]: That got lost when cleaning up copy'n'paste :) Re-added in 3068ef77ea66001efa9fff8d488caf726e890420.

File: pyproject.toml (hunk 2 above)
Comment (felixfontein) [reply pyproject.toml:20]: I would avoid tracking if it is not strictly necessary.

Comment (webknjaz) [reply pyproject.toml:20]: Well. Maybe it's not necessary but is useful. Let me expand that by tracking I don't mean tracking individuals but mostly something to help marketing and the docs people figure out where the users come from.

For example, here's what I've been doing in the past: https://github.com/ansible-collections/community.general/blob/ee23c26/.github/ISSUE_TEMPLATE/documentation_report.yml#L107 / https://github.com/ansible/ansible/blob/38a11c3/.github/ISSUE_TEMPLATE/config.yml#L5.

I think these are pretty harmless and don't contribute to the loss of privacy of individuals.

cc @gundalow do you have any thoughts on this?

Comment (gundalow) [reply pyproject.toml:20]: If Felix is happy, we can add it, otherwise I'm not going to push for it.

Comment (webknjaz) [reply pyproject.toml:20]: Okay, @felixfontein it's your call then. Feel free to merge.

Comment (felixfontein) [reply pyproject.toml:20]: @webknjaz which values should be in there?

I think I'll just merge this as-is and trigger the 0.1.0 release, then you can create a follow-up PR with the exact contents that should be in there.

Comment (webknjaz) [reply pyproject.toml:20]: I usually invent the values by example + context

**Pull request merged**